src/simulation.F90

```diff
@@ -516,7 +516,7 @@ function openmc_simulation_finalize() result(err) bind(C)
           n = size(results, 3)
           count_per_filter = size(results, 1) * size(results, 2)
           call MPI_TYPE_CONTIGUOUS(count_per_filter, MPI_DOUBLE, &
-               result_block, mpi_err)
+              result_block, mpi_err)
```

Comment:
Our style guide does say five spaces for a continuation line, so you should put the space back in here. This is one of the reasons the test suite is failing (there is a script that does a style check).